src/imagewriter.cpp

```diff
@@ -488,7 +488,7 @@ void ImageWriter::handleNetworkRequestFinished(QNetworkReply *data) {
     if (data->error() == QNetworkReply::NoError) {
         auto httpStatusCode = data->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
 
-        if (httpStatusCode >= 200 && httpStatusCode < 300) {
+        if (httpStatusCode >= 200 && httpStatusCode < 300 || httpStatusCode == 0) {
```

Comment:
I assume this works as-is, but would it be better to explicitly group the logical operators? i.e.
```
if ((httpStatusCode >= 200 && httpStatusCode < 300) || httpStatusCode == 0) {
```